config/routes.rb

```diff
@@ -25,7 +25,8 @@
   scope ":slug", module: "tenants", as: "tenant" do
```

Comment:
I feel like naming the slug ":slug" here might get confusing since we used ":tenant" elsewhere.